app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -3836,14 +3849,28 @@ class BrowserTabViewModel @Inject constructor(
     }
 
     fun onDuckChatMenuClicked() {
+        openDuckChat(pixelName = DuckChatPixelName.DUCK_CHAT_OPEN_BROWSER_MENU)
+    }
+
+    fun onDuckChatOmnibarButtonClicked(query: String?) {
+        openDuckChat(query = query)
+    }
+
+    private fun openDuckChat(pixelName: Pixel.PixelName? = null, query: String? = null) {
```

Comment:
Can we create a follow-up task to add tests for all the testable changes in this PR?